### DISCUSSION TITLE
Is it possible to use a custom obfuscator?

### DISCUSSION THREAD

**Discussion by hashimaziz1** [parcel-bundler/parcel#8210] [Q&A] [answered]
I need to use a more dedicated obfuscator with Parcel instead of Terser, such as [Obfuscator.io](https://obfuscator.io) - is it possible to do this, either by disabling Terser or otherwise?

**Comment by mischnic** [comment, marked as answer]
You can replace the terser optimizer with some other optimizer plugin using this parcelrc:
```json
{
	"extends": "@parcel/config-default",
	"optimizers": {
		"*.js": ["parcel-optimizer-that-you-want"]
	}
}
```
